test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -370,26 +459,30 @@ string ProtoConverter::testCode(unsigned _invalidLength)
 		</atLeastOneVar>
 		return 0;
 		)")
-		("parameterNames", dev::suffixedVariableNameList(s_varNamePrefix, 0, m_varCounter))
+		("argumentNames", m_argsCoder.str())
 		("invalidLengthFuzz", std::to_string(_invalidLength))
 		("isRightPadded", isLastDynParamRightPadded() ? "true" : "false")
 		("atLeastOneVar", m_varCounter > 0)
 		.render();
 }
 
-string ProtoConverter::helperFunctions()
+string ProtoConverter::testReturnDataFunction()
 {
-	stringstream helperFuncs;
-	helperFuncs << R"(
-	function bytesCompare(bytes memory a, bytes memory b) internal pure returns (bool) {
-		if(a.length != b.length)
-			return false;
-		for (uint i = 0; i < a.length; i++)
-			if (a[i] != b[i])
-				return false;
-		return true;
-	}
+	return Whiskers(R"(<?varsPresent>
+		(<varDecl>) = this.coder_returndata_external();
+<equality_checks></varsPresent>
```

Comment:
Indentation and line breaks.

Reply:
Hmm... I just realized that `<equality_checks>` is itself properly indented, which is why I didn't indent it here. Did I understand your comment correctly? Or were you referring to another indentation issue? :)

Reply:
Yeah ok... I'd have put a line break before ``</varsPresent>``, but it doesn't really matter too much (and might result in too many line breaks in the output), so it's fine.

Reply:
Ah I see. Yeah, I experimented a bit with that and came to the conclusion that it adds two newlines every time I do that. I somehow felt two lines is overkill, although I agree that this makes the Whiskers template less readable.

Reply:
Yeah, it's fine :-)!

Reply:
Just to be consistent, I think I'll add the new line before. I just realized that this is the only instance where I'm trying to save a new line when there are several new lines elsewhere :)

Reply:
Ok :-)!